.circleci/config.yml

```diff
@@ -79,7 +79,10 @@ jobs:
           command: |
             git clone "https://github.com/gohugoio/hugoDocs.git" ~/project
             rm ~/project/content/en/content-management/related.md
-      - hugo/hugo-build
+      - hugo/hugo-build:
```

Comment:
You can remove these changes in the file as this is the `hugo-build` command, not the `build` job. Since there's no build job in the config for testing and these params are passthrough to existing params, don't worry about making sure they are here.

Reply:
Got it. Removed in commit 344c2cd11f763d2160c3eb4df4c54a42d38a6688.